pkg/innerring/config/config.go

```diff
@@ -31,7 +32,7 @@ func NewGlobalConfigReader(cfg *viper.Viper, nm *netmapClient.Wrapper) *GlobalCo
 
 func (c *GlobalConfig) BasicIncomeRate() (uint64, error) {
 	value := c.cfg.GetUint64("settlement.basic_income_rate")
-	if value != 0 {
+	if isDebug() && value != 0 {
```

Comment:
Code is valid, but lets wrap value reading `cfg.Get` into
```
if isDebug() {
	...
}
```
statement in order to underline that we read config only in debug build.